src/index.ts

```diff
@@ -6,34 +6,21 @@ import { Readable } from 'readable-stream';
 
 if (!globalThis.Blob) {
   try {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-extraneous-dependencies
-    globalThis.Blob = require('formdata-node').Blob;
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    globalThis.Blob = require('node:buffer').Blob;
   } catch (e) {
-    throw new Error(
-      'Since you do not have the Blob API available in this environment you must install the optional `formdata-node` dependency.',
-    );
+    throw new Error('The Blob API is required for this library. https://developer.mozilla.org/en-US/docs/Web/API/Blob');
   }
 }
 
 if (!globalThis.File) {
   try {
-    // eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-extraneous-dependencies
-    globalThis.File = require('formdata-node').File;
+    // Node's native `fetch` implementation unfortunately does not make this API global so we need
```

Comment:
ah damn wait i think you answered this question i had lol — kinda confusing how `undici` is a separate npm package